src/sections/footer.html

```diff
@@ -1,3 +1,22 @@
-<div>
-    
-</div>
\ No newline at end of file
+<div class="flex justify-center pb-5 items-center sticky top-0 py-p-footer">
+    <img src="/img/red.webp" alt="The red vector">
+</div>
+<footer
+  class="flex flex-col text-black font-body font-extrabold space-y-60 px-2 lg:flex-row lg:items-start p-footer"
```

Comment:
Here; the font weight is font-medium so, can you please rectify that?

Reply:
chnages done